app/src/main/java/com/sapuseven/untis/adapters/infocenter/AbsenceAdapter.kt

```diff
@@ -26,14 +28,24 @@ class AbsenceAdapter(
 
 	override fun getItemCount() = absenceList.size
 
+	@SuppressLint("SetTextI18n")
 	override fun onBindViewHolder(holder: ViewHolder, position: Int) {
 		val absence = absenceList[position]
 
 		holder.tvTime.text = formatAbsenceTime(
 				UntisDate(absence.startDateTime).toDateTime().withZone(DateTimeZone.UTC),
 				UntisDate(absence.endDateTime).toDateTime().withZone(DateTimeZone.UTC)
 		)
-		holder.tvTitle.text = absence.absenceReason
+		holder.tvTitle.text =
+				if (absence.absenceReason.isNotEmpty())
+					absence.absenceReason.substring(0, 1).toUpperCase(Locale.getDefault()) + absence.absenceReason.substring(1)
```

Comment:
The suppression is needed because otherwise, Android Studio will complain about this line.

Reply:
Line 41

Reply:
Okay, I checked this and now it doesn't complain anymore. Last time it did.

Reply:
Maybe you fixed the warning unintentionally ;)
I'll delete this and merge